web/templates/base.html

```diff
@@ -6,7 +6,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 
-    <title>Code Thesaurus - {{ title }}</title>
+    <title>Code Thesaurus - {{ title|linebreaksbr }}</title>
```

Comment:
The title tag won't ever have a line break in it, so this isn't needed.

Reply:
Gotcha! Changed it back.

Reply:
By the way, this PR is a branch from this[PR](https://github.com/codethesaurus/codethesaur.us/pull/150). If you accept this pull, it will accept that one as well, but if you want to evaluate them separately you can do that too.